spec/system/admin_kudo_spec.rb

```diff
@@ -10,27 +10,30 @@
   let!(:employee1) { create(:employee) }
   let!(:employee2) { create(:employee) }
   let!(:admin_user) { create(:admin_user) }
-  let!(:kudo) { build(:kudo) }
```

Comment:
I am not sure - why you got rid of this?

Reply:
I had to leave this out because rubocop allows a maximum of 5 `let()` in spec. I didn't want to add an exception in rubocop so I entered kudo manually

Reply:
Please treat rubocop rather as suggestion in such case, not stick requirement - you can add exceptions , or simply increase the max value in rubocop config

Reply:
I changed the maximum allowed number of lets to 6. This is what we discussed on Tuesday with Oskar, when rubocop is right and when it can be changed :)